src/dag_algo/mod.rs

```diff
@@ -468,14 +483,29 @@ pub fn topological_generations(dag: &digraph::PyDiGraph) -> PyResult<Vec<NodeInd
     Ok(generations)
 }
 
-/// Return the topological sort of node indices from the provided graph
+/// Return the topological sort of node indices from the provided directed
+/// graph.
 ///
-/// :param PyDiGraph graph: The DAG to get the topological sort on
+/// Computes a topological ordering of the nodes in the given directed graph,
+/// ensuring that for every directed edge from node :math:`u` to node :math:`v`,
+/// node :math:`u` appears before node :math:`v` in the resulting sequence. This
+/// is particularly useful in scenarios such as task scheduling and dependency
+/// resolution, where certain tasks must be completed before others.
 ///
+///   >>> G = rx.PyDiGraph()
+///   >>> G.add_nodes_from(["A", "B", "C", "D", "E", "F", "G"])
+///   >>> G.add_edges_from_no_data([(0, 1),(1, 2), (2, 3), (3, 4), (5, 2), (6, 3)])
```

Comment:
ditto

Reply:
This is true for many of other examples and I'll be happy to update them later.

However, since this method returns a numerical list of indices `NodeIndices[6, 5, 0, 1, 2, 3, 4]`, I find it easier for the reader to match these numbers and the arguments to `add_edges_from_no_data`.